makeSlackDark.bat

```diff
@@ -1,3 +1,3 @@
 @echo off
-type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.3.1\resources\app.asar.unpacked\src\static\index.js
-type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.3.1\resources\app.asar.unpacked\src\static\ssb-interop.js
+type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.3.3\resources\app.asar.unpacked\src\static\index.js
+type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.3.3\resources\app.asar.unpacked\src\static\ssb-interop.js
```

Comment:
@Catscratch Just a note. Some people on Mac have noticed that newest versions of Slack will crash when starting if you modify the `ssb-interop.js` file. Speaking of which, it is no longer necessary to modify that file anymore. Changing `index.js` should be sufficient.

Reply:
Thanks for the hint. No problems on Windows so far. I'll give it a try on the next slack update.